apps/client/src/components/RegisterLookupSummary.tsx

```diff
@@ -14,23 +14,34 @@ import {
 import AddressLines from "./AddressLines";
 import ChangeAddressModal from "./Location/ChangeAddressModal";
 
-// @TODO: Convert this file to TS
-// @TODO replace unused props from RegisterLookupSummary in LocationResult
-const RegisterLookupSummary = ({
+// @TODO: can you make a props file for these kinds of props, @andre?
```

Comment:
Good question. Can you remove it from the source code however? I'll add it to the list of stuff to improve.